spec/factories/organizations.rb

```diff
@@ -10,9 +10,9 @@
     onboarding_success_heading { File.read(File.join('config', 'locales', 'onboarding', 'success_heading.txt')) }
     onboarding_success_text { File.read(File.join('config', 'locales', 'onboarding', 'success_text.txt')) }
     onboarding_data_protection_link { 'https://tactile.news/100eyes-datenschutz/' }
-    signal_server_phone_number { Faker::PhoneNumber.cell_phone }
-    whats_app_server_phone_number { Faker::PhoneNumber.cell_phone }
-    telegram_bot_username { Faker::Internet.username }
+    signal_server_phone_number { Faker::PhoneNumber.unique.cell_phone_in_e164 }
+    whats_app_server_phone_number { Faker::PhoneNumber.unique.cell_phone_in_e164 }
+    telegram_bot_username { Faker::Internet.unique.username }
```

Comment:
The first two lines are for consistency, even though we do not normalize nor do we validate on the organizaitons mode, yet. Maybe with the addition of our team adding these to the orgs, we should do so.

The last line seems to have gotten rid of the unique telegram bot errors. At least on 6 or so runs, it has not been encountered again.